### DISCUSSION TITLE
Removal of PR

### DISCUSSION THREAD

**Discussion by cweitkamp** [openhab/openhab-addons#14704] [Add-on Maintainers Discussions]
Dear all,

FYI

The contributor of PR https://github.com/openhab/openhab-addons/pull/10517 requested the removal of his PR because of a breach of contract with his employer.

I will pick it up later.

Best

**Comment by kaikreuzer** [comment]
What an antiquated mindset of his employer. Seems to be Ericsson, I wouldn't have thought of them being like that... Good reason for any skilled developer not to apply at Ericsson...
